Patch borrowed from CentOS mysql-stack.patch
Talk about silly byte-shaving ... the "stock" value does not work on s390x,
and probably not on some other platforms either.  Knock it up to twice the
pre-5.0.33 value to have some margin for future changes in compiler, glibc,
etc.  Suspect this number if "execution_constants" regression test fails.


diff -Naur mysql-5.0.38.orig/sql/mysql_priv.h mysql-5.0.38/sql/mysql_priv.h
--- mysql-5.0.38.orig/sql/mysql_priv.h	2007-03-20 15:12:46.000000000 -0400
+++ mysql-5.0.38/sql/mysql_priv.h	2007-04-16 16:09:39.000000000 -0400
@@ -175,7 +175,7 @@
  Feel free to raise this by the smallest amount you can to get the
  "execution_constants" test to pass.
  */
-#define STACK_MIN_SIZE          12000   // Abort if less stack during eval.
+#define STACK_MIN_SIZE          16384   // Abort if less stack during eval.
 
 #define STACK_MIN_SIZE_FOR_OPEN 1024*80
 #define STACK_BUFF_ALLOC	256	// For stack overrun checks
